CorpusSearch/Services/OpenDataLoader.cs

```diff
@@ -60,6 +62,34 @@ public static List<string> GetJsonPaths(LoadConfig lConfig)
                     : allFiles.ToList()
                 ;
         }
+
+        
+
+        public static async Task<List<(IDocument, DateTime)>> LoadLatestDocuments(WorkService workService)
+        {
+            var latestDocumentsFile = Path.Combine(
+                AppDomain.CurrentDomain.BaseDirectory,
+                "OpenData",
+                "newdocs.txt"
+            );
+             
+            var lines = await File.ReadAllLinesAsync(latestDocumentsFile);
+
+            var dateAndPath = lines.Select(x =>
+                (
+                    Date: DateTime.Parse(x[.."2024-02-27 04:50:38".Length]),
+                    Path: x["2024-02-27 04:50:38 ".Length..]
```

Comment:
I feel this is fairly readable, but it could do with a comment with a sample line